script/Ciel/Day 14/14_01A_CIEL14A_6.txt

```diff
@@ -146,22 +158,25 @@ In that case, I'm begging you, don't get yourself killed in a place like this.
 [sha:676172f669c68c91209f138422e362e9beaff459]{
 -- Page 217, Offset 38274.
 -- だから、謝るしかない。
-That's why I have no choice but to apologize.
+// Echo 38262.
+That's why I feel like I have to apologize.
 }
 [sha:7a7dd5ead8629f3f97e4a4859bb1d21c476efa1d]{
 -- Page 217, Offset 38275.
 -- 色々な事をしてあげたくて、けど結局は何一つ叶えられなかったというのなら。
-I wanted to do so many things for you. But in the end, I wasn't able to grant even a single wish.
+Even though I wanted to do so many things for her,
+ in the end,
+ I couldn't even grant that one wish.
 }
 [sha:76f73ff03352b8f3ec57ab164febb1f05f918a86]{
 -- Page 217, Offset 38276.
 -- せめて、最後に。
-So at the very least...
+That's why, as my final act―――
 }
 [sha:9a4cba27fd70bbd8d1d0ef9c4a1271f1a074ecd4]{
 -- Page 217, Offset 38277.
 -- 彼女を、こんなつまらない因縁から解放するぐらい、こなさないと。
-I want to set her free from this worthless fate.
+I want to at least set her free from this worthless fate.
```

Comment:
>I've gotta at least set her free from her wearisome fate.

Reply:
Don't love dropping the register with `gotta` in this moment

Reply:
Well, then you can uncontract it.
>I've got to at least